lib/quiet_quality/cli/arg_parser.rb

```diff
@@ -134,6 +135,16 @@ def setup_filter_messages_options(parser)
           read_tool_or_global_option(:filter_messages, tool, false)
         end
       end
+
+      def setup_logging_options(parser)
+        parser.on("-l", "--light", "Print aggregated results only") do
+          set_global_option(:logging, Logging::LIGHT)
+        end
+
+        parser.on("-q", "--quiet", "Don't print results, only return a status code") do
+          set_global_option(:logging, Logging::QUIET)
+        end
+      end
```

Comment:
What do you think about a `--logging LEVEL` flag as well? We're going to have more levels in #42 and #39, so it could just as easily be added in the next of those PRs.

And do you think `--light` is clear enough, or could it mean something other than logging?

Reply:
Would that be in addition to the `-l/--light` and `-q/--quiet` flags, or instead of? 

I think light is a fine name. `concise` might be good too.

Reply:
I was thinking "in addition" - we're going to end up with "quiet", "light", "normal", "verbose", and "debug" I think eventually :-)